build-aux/cargo.sh

```diff
@@ -9,10 +9,10 @@ export BUILDTYPE="$4"
 export APP_BIN="$5"
 
 
-if [[ $BUILDTYPE = "release" ]]
+if [ $BUILDTYPE = "release" ]
 then
     echo "RELEASE MODE"
-    cargo --offline --locked build --release --manifest-path \
+    cargo build --locked --release --manifest-path \
```

Comment:
Offline mode is necessary for flatpak build.

Perhaps do we handle a special case for flatpak?

Reply:
I added a special cases for flatpak (see latest commit), that should address this. I am not sure how to test it though.

Reply:
Great! Thanks so much!